documentation/index.html

```diff
@@ -425,6 +425,230 @@ <h3 class="content-heading">Getting Started</h3>
                     </pre>
                 </div>
                 <hr>
+                <!-- image component -->
+                <div class="image-container">
+                    <p class="content-sub_heading">Images</p>
```

Comment:
use semantic heading elements for headings

Reply:
Ok, I will update that